### PR TITLE
Added minikube to compat list + updated CTA for support 

### PR DESCRIPTION
**Pull request by macolso**
_index.md 
- Based on input from @rajatjindal, added a small disclaimer on how to ensure user is set up for success with minikube

Feedback.md
- Swapped out Discord link for CNCF slack link

### CODE REVIEW

File: content/en/docs/compatibility/_index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ See the following list of compatible Kubernetes distributions and platforms for
  - [Digital Ocean Kubernetes (DOKS)](https://www.digitalocean.com/products/kubernetes)
  - [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)
  - [k3d](https://k3d.io)
+ - [minikube](https://minikube.sigs.k8s.io/docs/) (explicitly pass `--container-runtime=containerd` and ensure you're on version `>= 1.33`)
```
Comment (endocrimes) [inline content/en/docs/compatibility/_index.md:18]:
```suggestion
 - [minikube](https://minikube.sigs.k8s.io/docs/) (explicitly pass `--container-runtime=containerd` and ensure you're on containerd version `>= 1.33`)
```

**Review by endocrimes — COMMENTED** [review]
Thanks!

File: content/en/docs/compatibility/_index.md (hunk 1 above)
Comment (rajatjindal) [reply content/en/docs/compatibility/_index.md:18]: this should be minikube version >= 1.33

Comment (endocrimes) [reply content/en/docs/compatibility/_index.md:18]: 🙈 (you're right - maybe we should reformat this to be a little clearer, given that it was easy to misread 😅)

Comment (macolso) [reply content/en/docs/compatibility/_index.md]: updated to minkube! thanks

**Review by rajatjindal — APPROVED** [review]

**Review by endocrimes — APPROVED** [review]

**Pull request merged**